tasks/src/main.rs

```diff
@@ -75,14 +76,15 @@ fn get_tests(verify: bool) -> Result<()> {
     let src_dir = "./src/parser/grammar.rs";
     let grammar = fs::read_to_string(src_dir)?;
     let tests = collect_tests(&grammar);
-    for (i, test) in tests.into_iter().enumerate() {
-        let path = format!("./tests/data/inline/test_{:02}.toml", i);
+    for (name, test) in tests.into_iter() {
+        let path = format!("./tests/data/inline/test-{}.toml", name);
```

Comment:
Let's check just in case that if we have a `foo` test file, then we have a `foo` inline test as well?